pkg/segment/results/segresults/segresults.go

```diff
@@ -413,6 +413,10 @@ func (sr *SearchResults) MergeRemoteRRCResults(rrcs []*utils.RecordResultContain
 	remoteCount uint64, earlyExit bool) error {
 	sr.updateLock.Lock()
 	defer sr.updateLock.Unlock()
+	if len(rrcs) != len(rawLogs) {
+		return fmt.Errorf("MergeRemoteRRCResults: rrcs and rawLogs length mismatch, len(rrcs): %v, len(rawLogs): %v, qid=%v", len(rrcs), len(rawLogs), sr.qid)
+	}
```

Comment:
always put qid at the first, see the other log msgs

Reply:
There are many error msgs that needs to be corrected in this file. For, now have cleaned up this method.